src/main/java/org/cactoos/scalar/NumberOf.java

```diff
@@ -65,49 +60,28 @@ public NumberOf(final String txt) {
         this(new TextOf(txt));
     }
 
+    public NumberOf(final Scalar<Long> lnm,
+                    final Scalar<Integer> inm,
+                    final Scalar<Float> fnm,
+                    final Scalar<Double> dnm) {
+        this(dnm);
```

Comment:
@ruca905 I think there was some kind of confusion about what `NumberOf` and `Sealed` were about… so, now that it's clearer to me, I think we should merge `Sealed` and `NumberOf` into `NumberOf`, for this:
- we move the old code based on multiple scalars from `NumberEnvelope` to `NumberOf`
- we move the constructors of `Sealed` to `NumberOf`
- we keep the text/string based constructor here and delegate to the scalar-based ones
- we delete `Sealed`